# A 'semantic' description of the space to be updated. The descriptions such as 'in_front_of'
# and 'on_top_of' must be specified in the model of the corresponding entity.

# Examples:
# "on_top_of(table-123)"        Updates table-123's pose and searches for objects on top
# "in_front_of(cabinet-7)"      Updates cabinet-7's pose and searches for objects in front
# "table-123"                   Updates table-123's pose  

string update_space_description

---
string[] new_ids      # ids of new entities
string[] updated_ids  # ids of updated entities
string[] deleted_ids  # ids of deleted entities

string error_msg      # Empty if no errors
